pmd-packager/upgrade.sh

```diff
@@ -1,5 +1,7 @@
-VERSION=$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | jq --raw-output ".tag_name" | sed 's:.*/::')
+VERSION=$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | grep '.tag_name' | sed 's:.*/::' | sed 's:",::')
 echo $VERSION
+echo -n $VERSION > pmd-packager/pmd-version.txt
```

Comment:
```suggestion
echo -n $VERSION > pmd-version.txt
```

Reply:
I think this change moves pmd-version.txt to the main directory. That's probably a good idea, as PMD is a major part of this project.

Reply:
This depends on what the current directory is.... if you run "npm run update-pmd", the script will change into "pmd-packager" first and then run "upgrade.sh".